identity-account/src/updates/update.rs

```diff
@@ -316,7 +317,7 @@ async fn insert_method_secret(
         )
       );
 
-      store.key_insert(did, location, private_key).await
+      store.key_insert(did, location, private_key).await.map_err(|e| e.into())
```

Comment:
Nit: prefer `Into::into`.

```suggestion
      store.key_insert(did, location, private_key).await.map_err(Into::into)
```